test/CMakeLists.txt

```diff
@@ -104,6 +106,20 @@ foreach(file ${files})
       CATCH_CONFIG_FAST_COMPILE
       $<$<CXX_COMPILER_ID:MSVC>:_SCL_SECURE_NO_WARNINGS>
     )
+    if (${CMAKE_VERSION} VERSION_LESS "3.8.0")
+        if (JSON_ForceCpp14)
+            target_compile_features(${testcase} PRIVATE cxx_binary_literals)
+        endif()
+        if (JSON_ForceCpp17)
+        message(FATAL_ERROR "Cannot force C++17 with this CMake version, you need at least version 3.8")
+        endif()    else()
```

Comment:
else should be on a new-line here, I think.

Reply:
True, sorry for the noise!